src/nft/components/collection/Card.tsx

```diff
@@ -595,7 +595,11 @@ const Suspicious = () => {
 const Pool = () => {
   return (
     <MouseoverTooltip
-      text={<Box fontSize="14">This item is part of an NFT liquidity pool. Price increases as supply decreases.</Box>}
+      text={
+        <Box fontSize="14">
```

Comment:
Should the text prop include markup (`<Box>`)?

Reply:
I think so because it lets us control the box size and the text prop takes in a ReactNode

Reply:
We could also swap fontSize '14' to be className `bodySmall` but this is non-blocking